app/controllers/application_controller.rb

```diff
@@ -1,2 +1,16 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+
+  before_action :update_allowed_parameters, if: :devise_controller?
```

Comment:
- [ ] Using `devise` gem here but not installed. Please add it to the Gemfile for proper configuration.

Reply:
I have fixed it. Thanks for your detailed review!!!